### ISSUE TITLE
[ANNOUNCEMENT] Update Required

### ISSUE THREAD

**Issue by ThatFruedDued** [ThatFruedDued/blooket-utility#41]
As you might have noticed, many things are not working right now in the utility. 99% of these issues can be fixed with an update. However, updating is very time consuming and it is not reasonable for me to do at the moment due to school taking up most of my free time. For now, I will be temporarily putting this project on hold. Here's what's going to happen:

1. I will not commit to this repository or make any updates.
2. Issues will be ignored because most of them are caused from the lack of the latest version.
3. I may look for a way around patching individual Blooket versions. Note that some features are extremely difficult to include by taking this route.

Sorry this had to happen, I'm just a bit overwhelmed right now. If you need a temporary alternative, I would suggest https://github.com/glixzzy/blooket-hack.

**UPDATE:**
I am currently testing a new loading system, which would no longer require manual updates from me. Please check out the beta by visiting https://blooket-utility.okr765.com (no bookmarklet required). You can also go to https://blooket-utility.okr765.com/conf to configure the utility. I have not added pre-installed prefs as I have been attempting to move towards a more modular approach with the injectable system. **Please note that injectables designed for the old Blooket Utility will not work with the new one.** As far as I know, this only includes one injectable (https://thatfrueddued.github.io/blooket-utility-injectables/injectables/glixzzy.js). To add a new injectable, use the `injectable.json` file (like https://thatfrueddued.github.io/blooket-utility-injectables/injectables/customMaps/injectable.json). That should be all for this update!

**Comment by Fe4RLess1457** [comment]
Ok, do you know if you will ever start this again because yours is so much better

**Comment by wolfgoo** [comment]
Yes, hack glixzzy did also have bug.

**Comment by Fe4RLess1457** [comment]
I will pay you to update because the other one was blocked by my admin

**Comment by SwingWitMe** [comment]
same @Fe4RLess1457  the glixzzy hack is blocked now i dont really use this one only because it blocks up the screen (the mod menu) and i dont know how to minimize it

**Comment by wolfgoo** [comment]
???

**Comment by wolfgoo** [comment]
glixzzy hack isn't blocked now.

**Comment by SwingWitMe** [comment]
@wolfgoo fr its not blocked anymore let me try it

**Comment by SwingWitMe** [comment]
Damn jit its still blocked 
![Screenshot 2021-11-16 08 20 11](https://user-images.githubusercontent.com/94390684/141992862-632e659b-da5d-4a57-beee-bd7586311b83.png)

**Comment by wolfgoo** [comment]
:<

**Comment by wolfgoo** [comment]
But now I use it not block

**Comment by ShiixeN** [comment]
glixzzy's hacks dont even work anymore. great. now we have no hacks.

**Comment by wolfgoo** [comment]
ya

**Comment by wolfgoo** [comment]
very very bad

**Comment by wolfgoo** [comment]
3:

**Comment by ThatFruedDued** [comment]
Fixed! Check the new README for more information.
